### ISSUE TITLE
Fix mesecons lightstone on/off contrast

### ISSUE THREAD

**Issue by BruceMustache** [MysticTempest/REFI_Textures#3]
I like the design of lightstones in refi but is tiny difficult to know if it's on or off as comparing with the default texture:

REFI lightstone:
https://vimeo.com/user119376123/review/440307279/89b14f4704

default:
https://vimeo.com/user119376123/review/440308761/d464cb366a

**Comment by MysticTempest** [comment]
Alrighty, I'll see about making the lit versions brighter, and tweak the visuals a bit to better signify the difference between on & off.

**Comment by MysticTempest** [comment]
Here's a sample of my brighter versions; with an extra lit edge on the stone to signify being on:  
[mesecons_lightstone.zip](https://github.com/MysticTempest/REFI_Textures/files/4969789/mesecons_lightstone.zip)


And, a preview of night/day time viewing. But, my brightness settings may differ a bit from yours. So give them a try on your end, and see if they're acceptable.  
![new__brighter_mesecons_lightstone_on](https://user-images.githubusercontent.com/32621758/88352938-4b599b80-cd21-11ea-9eb6-ee0be602d282.png)

**Comment by BruceMustache** [comment]
It's so good now! 
Can you change the white lightstone texture to be white? 
Awesome work mystic, best minetest texture! 😁

video: https://streamable.com/i723cv

**Comment by MysticTempest** [comment]
> It's so good now!
> Can you change the white lightstone texture to be white?
> Awesome work mystic, best minetest texture! grin
> -snip screenshot-
> video: https://streamable.com/i723cv ( can you see that video?)

Thanks for the feedback & support, BruceMustache!  Your picture, and video works. 

I changed the white lightstones to be on the whiter side; instead of the partial teal that it was. And, adjusted the gray lightstones a little as well.

[mesecons_lightstone.zip](https://github.com/MysticTempest/REFI_Textures/files/4975243/mesecons_lightstone.zip)

**Comment by BruceMustache** [comment]
Nice! I like the new "white" and "dark grey" 
But I think  "grey" is very similar to "white"
![screenshot_20200725_182210](https://user-images.githubusercontent.com/46538696/88466554-f72af480-cea3-11ea-8153-5a1a47b09d68.png)

**Comment by MysticTempest** [comment]
> Nice! I like the new "white" and "dark grey"
> But I think "grey" is very similar to "white"
> I suggest you to use the texture of darkgrey in grey and make darkgrey more black
> ![screenshot_20200725_182210](https://user-images.githubusercontent.com/46538696/88466554-f72af480-cea3-11ea-8153-5a1a47b09d68.png)

Thanks for the suggestion. These updated grays should fix that issue.
[mesecons_lightstone.zip](https://github.com/MysticTempest/REFI_Textures/files/4977299/mesecons_lightstone.zip)

**Comment by BruceMustache** [comment]
Thanks for the update. The gray and darkgray are beautiful
If you want I suggest to make blue more blue (sorry I missed that in previous comment)
And change magenta color for something like:
![magenta](https://user-images.githubusercontent.com/46538696/88479450-a1972c00-cf25-11ea-9b96-ecb83f769b9d.png)
If you want to compare REFI with normal texture:
1. **Off**:
   * REFI: ![refi_off](https://user-images.githubusercontent.com/46538696/88479538-2b46f980-cf26-11ea-80e3-ad9d952001a6.png)
   * Normal: ![normal_off](https://user-images.githubusercontent.com/46538696/88479541-326e0780-cf26-11ea-94bc-50b7c1f5e308.png)
2. **On**:
   * REFI: ![refi_on](https://user-images.githubusercontent.com/46538696/88479544-369a2500-cf26-11ea-9931-fb2cabad34e4.png)
   * Normal: ![normal_on](https://user-images.githubusercontent.com/46538696/88479546-3d289c80-cf26-11ea-8421-acc7606caef1.png)

**Comment by MysticTempest** [comment]
To be quite honest; I'm not too fond of the colors used by the Mesecon Lightstones' default textures.
So, I don't plan on changing the rest of these colors. They match well with the the palette I've chosen for all the other colored nodes.
But, I've tweaked the magenta lightstone to be a bit closer to my other Magenta-colored nodes.

I do appreciate the help with the rest. 
I just pushed the changes with the latest commit:
https://github.com/MysticTempest/REFI_Textures/commit/55a581601057a497f91bf293f4c70f2dc15621a0
